kotlinx-coroutines-core/common/src/CoroutineScope.kt

```diff
@@ -117,13 +123,30 @@ public fun MainScope(): CoroutineScope = ContextScope(SupervisorJob() + Dispatch
 /**
  * Returns `true` when the current [Job] is still active (has not completed and was not cancelled yet).
  *
- * Check this property in long-running computation loops to support cancellation:
+ * A coroutine cancallation [is cooperative](https://kotlinlang.org/docs/cancellation-and-timeouts.html#cancellation-is-cooperative)
+ * and normally, the cancellation machinery happens when the coroutine *suspends*, for example,
+ * when trying to read from a [channel][kotlinx.coroutines.channels.Channel] that is empty.
+ *
+ * Sometimes, a coroutine does not need to do any of such operations, but still wants to be a cooperative
+ * and respect cancellation.
+ *
+ * [isActive] property is inteded to be used for scenarios like this:
```

Comment:
```suggestion
 * The [isActive] property is inteded to be used for scenarios like this:
```
[— There's a uniquely named property that you want.
— Which property is that?
—] The `isActive` property.